criu/cr-dump.c

```diff
@@ -37,13 +37,17 @@
 #include "crtools.h"
 #include "cr_options.h"
 #include "servicefd.h"
-#include "string.h"
 #include "ptrace-compat.h"
 #include "util.h"
 #include "namespaces.h"
 #include "image.h"
 #include "proc_parse.h"
 #include "parasite.h"
+/* string.h is included after parasite.h because the libbsd headers define
+ * __has_include as the constant 1.
+ * https://github.com/checkpoint-restore/criu/issues/2036
+ */
+#include "string.h"
```

Comment:
This solution seems too fragile for me. At some point we might need to include "linux/rseq.h" somewhere else, not only inside parasite.h.